src/app/manage-deposits/manage-single-deposit/manage-single-deposit.component.ts

```diff
@@ -55,7 +55,9 @@ export class ManageSingleDepositComponent implements OnInit {
                     return this.popupService.new({
                         type: 'error',
                         title: 'Not found',
-                        text: 'Deposit transaction with this reference code was not found.'
+                        text: 'Deposit transaction with this reference code was not found.',
+                        customClass: 'popup-error',
+                        position: 'top'
```

Comment:
I've seen that you are duplicating `customClass` and `position` all the time. 

I would recommend that, inside `this.popupService.new()` method, first create a "template" object containing these defaults:

```
const defaults: <Type> = {
  customClass: 'popup-error',
  position: 'top'
}
```

Then you return a new object by spreading:

```
return {
   ...defaults,
   ...objectFromParameter
}
```

Reach me if you don't get it.

Reply:
I added a new method in the service that helps us add class depending on the popup type defined in options.
Check it out and tell me what you think 😄